fuzz/Cargo.toml

```diff
@@ -0,0 +1,24 @@
+[package]
+name = "tiled-fuzz"
+version = "0.0.0"
+publish = false
+edition = "2018"
```

Comment:
Any reason to pick the 2018 edition specifically? Is it required for the fuzz crate?

Reply:
It was the version `cargo +nightly fuzz init` decided to use

Reply:
Alright, let's leave it that way just in case then.